src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.SslProtocols.cs

```diff
@@ -120,11 +120,19 @@ public void DisabledProtocols_SetSslProtocols_ThrowsException(SslProtocols disab
         [OuterLoop("Avoid www.ssllabs.com dependency in innerloop.")]
         [Theory]
         [MemberData(nameof(SupportedSSLVersionServers))]
-        public async Task GetAsync_SupportedSSLVersion_Succeeds(string name, string url)
+        public async Task GetAsync_SupportedSSLVersion_Succeeds(SslProtocols name, string url)
         {
-            using (var client = new HttpClient())
+            using (HttpClientHandler handler = new HttpClientHandler())
             {
-                (await client.GetAsync(url)).Dispose();
+                // Default protocol selection is always TLSv1 on Centos7 libcurl 7.29.0
```

Comment:
Would you consider slightly better wording? (it would help clarify it at least to me :))
"Set specific protocol we want to test to override the default protocol selection which is always TLSv1 on Centos7 libcurl 7.29.0"

Maybe it would be better to move the comment into the if-statement?